lego/apps/surveys/notifications.py

```diff
@@ -0,0 +1,32 @@
+from lego.apps.notifications.constants import SURVEY_CREATED
+from lego.apps.notifications.notification import Notification
+
+
+class SurveyNotification(Notification):
+
+    name = SURVEY_CREATED
+
+    def generate_mail(self):
+        survey = self.kwargs['survey']
+
+        return self._delay_mail(
+            to_email=self.user.email,
+            context={
+                'name': self.user.full_name,
+                'survey': survey.id,
+                'event': survey.event.title,
+                'active_from': survey.active_from,
+            },
+            subject=f'Undersøkelse etter Abakus-arrangement',
```

Comment:
Nitpicking, but maybe `f'Spørreundersøkelse for {survey.event.title}'` or something like that. Could be weird with certain event-names, but I think having the event title in the subject makes sense.